packages/core/contracts/oracle/implementation/OptimisticOracle.sol

```diff
@@ -545,7 +546,7 @@ contract OptimisticOracle is OptimisticOracleInterface, Testable, Lockable {
      * @return the stampped ancillary bytes.
      */
     function stampAncillaryData(bytes memory ancillaryData, address requester) public pure returns (bytes memory) {
```

Comment:
I didn't want to change this public method name yet in case client bots call it directly. Will do later.